Xamarin.Forms.Platform.Android/StepperRendererManager.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.ComponentModel;
+using Android.Views;
+using AButton = Android.Widget.Button;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public static class StepperRendererManager
+	{
+		public static void CreateStepperButtons<TButton>(IStepperRenderer renderer, out TButton downButton, out TButton upButton)
+			where TButton : AButton
+		{
+			downButton = (TButton)renderer.CreateButton();
+			downButton.Id = Platform.GenerateViewId();
+			downButton.Focusable = true;
+			downButton.Text = "-";
```

Comment:
Should we be using the minus sign (U+2212) here instead of the hyphen-minus (U+002D) here?
That is, should we be using "−" (minus sign) instead of "-" (the one you get from your keyboard)?

Reply:
I like that.

Reply:
Going to try full-width characters ( ＋ /  － )

Reply:
I'm down with the full-width hyphen-minus.